### PR TITLE
Day18 don 2player Functionality

### PR DESCRIPTION
**Pull request by wrathofdon**

### CODE REVIEW

File: js/init.js [hunk 1]
```diff
@@ -3,51 +3,96 @@
...
-  new Tile('kitten_08.jpg'),
-  new Tile('kitten_01.jpg'),
-  new Tile('kitten_02.jpg'),
-  new Tile('kitten_03.jpg'),
-  new Tile('kitten_04.jpg'),
-  new Tile('kitten_05.jpg'),
-  new Tile('kitten_06.jpg'),
-  new Tile('kitten_07.jpg'),
-  new Tile('kitten_08.jpg')
+  new Tile('Christian Bale', 'bale_1.jpg', heroDeserved, upsideDown, upsideDown),
```
Comment (ghost) [inline js/init.js:66]: Hilarious!

**Review by ashkrao — COMMENTED** [review]
Nice pictures!

File: mvp-user-stories.md [hunk 2]
```diff
@@ -26,6 +26,8 @@ Same
 
 ## Stretch goals:
 
+* As a customer, Chris wants to make it easier for his fans to tell the difference between him and other people with the same name.
```
Comment (ghost) [inline mvp-user-stories.md:29]: Maybe discuss implementation order of stretch goals.

File: css/style.css [hunk 3]
```diff
@@ -23,7 +23,11 @@
   text-align: center;
 }
 
-
+.tile {
```
Comment (ghost) [inline css/style.css:26]: TAKE

File: js/init.js [hunk 4]
```diff
@@ -3,51 +3,96 @@
...
 var tilesRemain = gridSize;
 // the total number of turns for the entire game
-var startTurns = gridSize * 2;
+// var startTurns = gridSize * 2;
+var startHP = 150;
 // array of flipped Tiles
 var flipped = [];
-
+// array of strings for at the end
+var exposition = [];
```
Comment (ghost) [inline js/init.js:15]: Change name. Needs to be more explicit

File: js/init.js [hunk 5]
```diff
@@ -113,27 +178,52 @@ function shuffle(array) {
...
 }
 
-function removeTurn(){
-  currentPlayer.turns--;
-}
 
-function addPoints(){
-  currentPlayer.points++;
+// in case we ever need to reload tiles, such as when re-loading from saved states
+function refreshTiles () {
```
Comment (ghost) [inline js/init.js:203]: TAKE - but needs changed name

**Pull request closed without merging**